Automattic/jetpack
# CRM: UI Cohesiveness Make Naming Consistent

This is the feature branch for the Make Naming Consistent for the Cohesiveness project.

## Proposed changes:
* The following issues are part of this Epic:
* https://github.com/Automattic/zero-bs-crm/issues/2761
* https://github.com/Automattic/zero-bs-crm/issues/2786
* https://github.com/Automattic/zero-bs-crm/issues/2788
* https://github.com/Automattic/zero-bs-crm/issues/2910

### Other information:


## Jetpack product discussion


pbhBOz-2Wj-p2


## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Make sure all merged PRs merged into this branch were properly reviewed and approved.
* Make sure this branch doesn't have any conflicts with `trunk`
